composer.json

```diff
@@ -10,12 +10,12 @@
         }
     ],
     "require": {
-        "php": "~8.1.0 || ~8.2.0",
+        "php": "~8.2.0",
```

Comment:
This is harsh: do you plan to do the same for other packages like `lcobucci/jwt` too?